### ISSUE TITLE
learning rate of direct pretraining for retinanet

### ISSUE THREAD

**Issue by belisarius0** [wxzs5/direct-pretraining#6]
Isn't the learning rate of direct-pretraining for RetinaNet 0.08 ?
Wrt the paper, 0.04 is the setting on the less data.

If I'm mistaken, my apologies.
